teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<ul><li>Spaces before and after =, but I think all this will be caught by the <code>black-nb</code> lateron.</li></ul><p> </p><p> </p><ul><li>What is <code>pd.to_numeric(klifs_pocket_df['residue.id'])</code> doing?</li></ul><p>The residue IDs are strings and you want to convert them to float/int?</p><p>I usually use</p><p><code>klifs_pocket_df = klifs_pocket_df.astype({'residue.id': int})</code></p><p>for that.</p><p> </p><p> </p><ul><li>Check the # at the end of the 5th line.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<ul><li>Space, done.</li><li>To_numeric conversion, yes that's what it does, but your version is way more elegant, adapted.</li><li>Deleted # </li></ul>